tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -582,6 +582,10 @@ def test_ended(self, test, results):
         if status == "CRASH":
             self.browser.log_crash(test.id)
 
+        if not file_result.extra:
+            file_result.extra = { "test_timeout": test.timeout }
+        else:
+            file_result.extra.test_timeout = test.timeout
```

Comment:
Does this work? I would expect it to be `file_result.extra["test_timeout"]` unless `extra` isn't a dictionary for some reason.